metatron/src/main/java/uk/gov/ida/eidas/metatron/domain/KeyStoreModule.java

```diff
@@ -48,11 +49,10 @@ public KeyStore deserialize(JsonParser jp, DeserializationContext ctxt) throws I
             try {
                 byte[] decoded = Base64.getDecoder().decode(node.asText());
                 keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
-                keyStore.load(new ByteArrayInputStream(decoded), "marshmallow".toCharArray());
+                keyStore.load(new ByteArrayInputStream(decoded), "".toCharArray());
             } catch (KeyStoreException | IOException | NoSuchAlgorithmException | CertificateException e) {
                 keyStore = null;
```

Comment:
Could this be moved to where the keystore is declared on line 48 (`KeyStore keyStore = null`) to save having this line here?

Reply:
I guess it could, but even if we were paying for storage by the byte I feel like it's worth extra few characters to point out that we're deliberately returning a null keyStore (which I'm not in love with) rather than that we're ignoring the fact that it failed to create one.